docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/{{ cookiecutter.module_name }}/utils.py

```diff
@@ -5,8 +5,8 @@ def get_site_menu():
     """Taken from the CALES app
     https://github.com/datamade/ucb-cales/pull/120/files
     #diff-b59096721a79856b109fac1104ad803e71e68fa4fe2660bf850048a8afe66840"""
-    from .models import BasePage
-    site_home = Page.objects.type(BasePage).first()
+    from .models import HomePage
```

Comment:
this is a fix we should make on other sites that use this `get_site_menu()` function. I'm surprised it hasn't caused any issues yet